opm/simulators/utils/readDeck.cpp

```diff
@@ -299,7 +274,7 @@ void readDeck(int rank, std::string& deckFilename, std::unique_ptr<Opm::Deck>& d
 #if HAVE_MPI
         MPI_Finalize();
 #endif
-        exit(EXIT_FAILURE);
+        exit(1);
```

Comment:
why was it changed back to 1?

Reply:
There were two of those and the changed one got lost when refactoring. Anyway, fixed.